low-level/files/opt/seagate/sspl/bin/generate_resource_health_view/resource_health_view

```diff
@@ -1134,5 +1134,6 @@ if __name__ == "__main__":
     parser.add_argument("--path", metavar="<path> eg. --path '/tmp/sspl/'", help="Health view\
             schema destination path (Note: Path need to be already exist)")
 
-    sys.exit(SSPLHealthView().run(parser))
+    SSPLHealthView().run(parser)
+    sys.exit(file_store.chown(f'/var/{PRODUCT_FAMILY}/sspl/', user='sspl-ll', group='sspl-ll', recursive=True))
```

Comment:
there must be some python function to recursively change permissions for given dir... not sure if we need to write this of our own?

Reply:
I agree

Reply:
There is no function for recursive permissions